pypesto/optimize/ess/sacess.py

```diff
@@ -445,10 +445,15 @@ def submit_solution(
                 # reject solution
                 self._rejections.value += 1
 
+                rel_change = (
+                    abs(abs_change / self._best_known_fx.value)
+                    if self._best_known_fx.value != 0
+                    else np.nan
+                )
                 self._logger.debug(
                     f"Rejected solution from worker {sender_idx} "
                     f"abs change: {abs_change} "
```

Comment:
Why is relative change formatted to at most four digits but absolute change not?

Reply:
:man_shrugging: 
Couldn't decide on what's reasonable yet.